packages/utils/parcelforvscode/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017-present Devon Govett
```

Comment:
Hmm we should probably change this to "Parcel Contributors" or something at some point.